053-SentinelAutomatedResponse/README.md

```diff
@@ -0,0 +1,45 @@
+# What The Hack - Sentinel Automated Response
+
+## Introduction
+This Hack will introduce you to Microsoft Sentinel by helping you implement and explore the core functionality of Microsoft's SIEM/SOAR platform.
+
+## Learning Objectives
+In this hack you will learn how to architect Sentinel, start ingesting data, use the Watchlists feature, create a custom alert and incident.  Finally you will
+learn how to add some automation to manage that incident.  
+
+1. Decide on Sentinel Architecture
+2. Install the agent and start recieving logs
+3. Create a watchlist
+4. Create a custom alert and generate an incident
+5. Implement a logic app to automatically close the alert
+
+## Challenges
+- Challenge-01: **[Architecture, Agents, Data Connectors and Workbooks](Student/Challenge-01.md)**
+   - Understand the various architecture and decide on the appropriate design based on the requirements. Install the appropriate data connector to import Windows security events and validate Log Analytics data ingestion.
```

Comment:
Try to reduce the challenge descriptions in the table of contents to just a single sentence. if possible.